components/script/dom/bindings/trace.rs

```diff
@@ -582,6 +584,13 @@ where
     }
 }
 
+unsafe impl JSTraceable for Rotation3D<f64> {
+    #[inline]
+    unsafe fn trace(&self, _trc: *mut JSTracer) {
+        // Do nothing
+    }
+}
+
```

Comment:
Nit: `unsafe_no_jsmanaged_fields!`?

Reply:
IIRC it doesn't work with generics that well? I'll try.